src/bindings/fluid_cmd.c

```diff
@@ -397,7 +398,11 @@ fluid_source(fluid_cmd_handler_t* handler, const char *filename)
     return file;
   }
   fluid_shell_init(&shell, NULL, handler, file, fluid_get_stdout());
-  return fluid_shell_run(&shell);
+  result = fluid_shell_run(&shell);
+
+  close(file);
```

Comment:
Shouldnt that be `_close()` for WIN32?

Reply:
Actually, it compiles anyways with oldnames activated, but you are right, probably it's better to use directly the name of the dllimport. I will change it.